rbac/common/util/__init__.py

```diff
@@ -31,3 +31,37 @@ def get_environment(name, default):
     if value is None or not value:
         return default
     return value
+
+
+def bytes_from_hex(value):
+    """A version of bytes.fromhex that will return None if
+    passed None instead of an expected str exception"""
+    if not value:
+        return None
+    return bytes.fromhex(value)
+
+
+def has_duplicates(anylist):
+    """Returns whether anylist has duplicates"""
```

Comment:
This function appears to be comparing the lengths of two lists - not checking for duplicates. If there are duplicates, then part of the logic to determine their existence is happening somewhere else. Also, it is performing validation. That could happen externally before the primary goal happens here.

Reply:
It's comparing the length of a list to the length of the set of the list. This works fine. Do you have another method? The function expects only lists as input params. It's a generic utility function, thus it validates its inputs.

Reply:
The quickest change would be to rename it has_same_length()